### ISSUE TITLE
final report --peer review

### ISSUE THREAD

**Issue by ann-qin-lu** [yuanchengl/ORIE4741-Seimic-Analysis-Project#8]
The topic is extremely interesting. And  I am amazed by the idea of making the prediction without solving the differential equations. The report is well-structured, but I have to admit that I don't quite follow the part of some data description.(Actually I am confused. What's n stands for? Does each data point consist n*p matrix? Then what's the size of dataset?) Somethings might need for further efforts:
1. try some dimensional reduction methods, which might further help saving computation.
2. It seems to have problems of overfitting with the model, but the error rate in test set seems quite well. However, the error rate in regression part seems a little confusing. What's the standard level of the mean squared error?
3. Try more nonlinear models. Maybe the neural network model might help, since the underlying problem seems to be non-linear.
 Overall, the project seems to be very exciting. Hope your model would be implemented someday!
